src/ripple/app/misc/FeeEscalation.md

```diff
@@ -118,12 +133,12 @@ the fee level is scaled appropriately.
 Examples, assuming a base fee of 10 drops:
 
 1. A single-signed [reference transaction](#reference-transaction)
-with `Fee=20` will have a fee level of 512.
+with `Fee=20` will have a fee level of `20 * 256 / 10 = 512`.
 2. A multi-signed transaction with 3 signatures (base fee = 40 drops)
-and `Fee=60` will have a fee level of 384.
+and `Fee=60` will have a fee level of `60 * 256 / (4 * 10) = 384`.
 3. A hypothetical future non-reference transaction with a base
 fee of 15 drops multi-signed with 5 signatures and `Fee=90` will
-have a fee level of 256.
+have a fee level of `90 * 256 / (15 * 6) = 256`.
```

Comment:
Nitpick: Use a base of 10 as the header mentions and reorder the divisor to be consistent.
eg. `90 * 256 / (6 * 10) = 384`


It may not be clear to the reader why the base fee is multiplied by one greater than the number of signatures. Another possible way of writing it might be:
60 * 256 / ((1+3) * 10) = 384
90 * 256 / ((1+5) * 10) = 384